ui/app/app.js

```diff
@@ -65,6 +65,7 @@ export default class App extends Application {
         ],
         externalRoutes: {
           secrets: 'vault.cluster.secrets.backends',
+          list: 'vault.cluster.secrets.backend.list-root',
```

Comment:
I think something like `secretsList` might make it more clear where this is linking too. Or maybe even better `secretsListRoot` since secrets list makes me think of the backends route which lists all the secrets mounts.